base16-manager

```diff
@@ -190,7 +190,7 @@ set_theme() {
         ;;
       "nicodebo/base16-zathura")
         set_generic "${package}" "${theme}" "#" ' ' \
-          "$HOME/.config/zathura/config"
+          "$HOME/.config/zathura/zathurarc"
```

Comment:
According to the [official docs](https://pwmt.org/projects/zathura/documentation/), the configuration file should be called `zathurarc`.